Sources/DeploymentTargetAWSLambdaRuntime/LambdaIntegration.swift

```diff
@@ -0,0 +1,128 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+//
+//
+// This code is based on the vapor-aws-lambda-runtime project: https://github.com/vapor-community/vapor-aws-lambda-runtime
+//
+// SPDX-FileCopyrightText: TODO @Paul what should I put here?
```

Comment:
@PSchmiedmayer question: I'm referencing the [vapor-aws-lambda-runtime](https://github.com/vapor-community/vapor-aws-lambda-runtime) project here, since the code of the Lambda integration is based on that. But I'm not really sure what I should use as the FileCopyrightText? Should this just be "the vapor-aws-lambda authors" or something like that? They don't have license headers in their files, otherwise i'd just have used that as a reference point...

Reply:
Yes, I think that "the vapor-aws-lambda authors" sounds right. The license file also just contains a placeholder and it gives credit to the right persons.